applications/ShapeOptimizationApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -49,16 +49,19 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
     // ================================================================
     class_<MapperVertexMorphing >(m, "MapperVertexMorphing")
         .def(init<ModelPart&, Parameters>())
+        .def("InitializeMapping", &MapperVertexMorphing::InitializeMapping)
```

Comment:
If there is an initialize, ther should be also finalize, even if it is empty.

Reply:
It is possible to use the base class for eyposing the functions to python, so we don`t need to copy it below.

Reply:
Done. In doing so, I also noticed that we never derived the individual mappers from the base class in the first place.